libs/components/indicators/src/lib/modules/tokens/tokens.component.ts

```diff
@@ -189,29 +188,30 @@ export class SkyTokensComponent implements OnDestroy {
       this.focusIndexUnderRange.emit();
     }
 
-    this._activeIndex = value;
+    this.#_activeIndex = value || 0;
   }
 
-  /**
-   * internal
-   */
-  public trackTokenFn: (index: number, item: SkyToken) => any;
+  public trackTokenFn: TrackByFunction<SkyToken>;
+
+  #_disabled = false;
+  #_dismissible = true;
+  #_focusable = true;
+  #_tokensOrDefault: SkyToken[] = [];
+  #_displayWith = 'name';
 
   @ViewChildren(SkyTokenComponent)
-  private tokenComponents: QueryList<SkyTokenComponent>;
+  public tokenComponents!: QueryList<SkyTokenComponent>;
```

Comment:
I brought this up during standup a few weeks ago (using non-null assertions on view children) and we agreed we should be using `| undefined` since they're technically undefined in the constructor. I'm personally okay using non-null assertions on view children, but we should probably be consistent. Which do you prefer?

Reply:
Fixed.